server/src/main/java/org/candlepin/audit/EventFactory.java

```diff
@@ -210,22 +213,23 @@ public Event subscriptionExpired(Subscription subscription) {
 
     public Event complianceCreated(Consumer consumer,
         Set<Entitlement> entitlements, ComplianceStatus compliance) {
-        StringBuilder eventDataBuilder = new StringBuilder()
-            .append("{\"consumer\": ")
-            .append("{\"uuid\": \"")
-            .append(consumer.getUuid())
-            .append("\"}")
-            .append(", \"status\": \"")
-            .append(compliance.getStatus())
-            .append("\"")
-            .append("}");
+        Map<String, String> eventData = new HashMap<String, String>();
+        eventData.put("consumer_uuid", consumer.getUuid());
+        eventData.put("status", compliance.getStatus());
+        String eventDataJson = "";
+        try {
+            eventDataJson = mapper.writeValueAsString(eventData);
+        }
+        catch (JsonProcessingException e) {
+            e.printStackTrace();
```

Comment:
Why is this eating the exception? Also, if it is to log the issue, it should be using our standard logger.